contracts/mocks/Claims/CLMockUnknownNFT.sol

```diff
@@ -13,4 +13,9 @@ contract CLMockUnknownNFT is ERC721 {
     _mint(to, tokenId);
   }
 
+  function tokenURI(uint id) public view override returns (string memory) {
+    id; 
```

Comment:
Since this is a mock and we don't need var names explicitly here, we can remove the variable name from the argument list to silence the error:
`function tokenURI(uint id)` => `function tokenURI(uint)`